internal/pkg/cli/pipeline_init.go

```diff
@@ -150,11 +150,25 @@ func newInitPipelineOpts(vars initPipelineVars) (*initPipelineOpts, error) {
 
 // Validate returns an error if the flag values passed by the user are invalid.
 func (o *initPipelineOpts) Validate() error {
-	// TODO add validation for flags
-	if o.appName == "" {
-		return errNoAppInWorkspace
-	}
+	// May consider validating githubAccessToken and gitBranch flag values in the future.
```

Comment:
nit: let's remove this TODO if we don't have immediate plans to support it. It may add confusion in the short term for other developers.